lua_modules/ftp/README.md

```diff
@@ -0,0 +1,64 @@
+# FTPServer Module
+
+This is a Lua module to access the SPIFFS file system via FTP protocol. 
+FTP server uses only one specified user and password to authenticate clients.
+All clients need authentication - anonymous access is not supported yet.
+All files have RW access.
+Directory creation do not supported because SPIFFS do not have that.
```

Comment:
While this is technically correct you might want to mention that you can mimic directories  by prefixing file names.